test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedCount.java

```diff
@@ -26,6 +26,8 @@
 import org.openjdk.jmh.annotations.BenchmarkMode;
 import org.openjdk.jmh.annotations.Fork;
 import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
```

Comment:
Should there have been annotations for Fork to go with this import?
Importing Measurement again is a dup.

Reply:
Good eye, this one has some mistake, I will fix these and get a new time estimate overnight.